### PR TITLE
improve vibration

### PR DESCRIPTION
**Pull request by pelikhan**
Vibration firmware should have a concept of "auto-off"

### CODE REVIEW

File: services/vibration.md [hunk 1]
```diff
@@ -10,3 +10,12 @@ Control a vibration motor.
 
 Rotation speed of the motor.
 If only one rotation speed is supported, then `0` shell be off, and any other number on.
+
+## Commands
+
+    command vibrate @ 0x80 {
```
Comment (mmoskal) [inline services/vibration.md:16]: maybe we should really do vibration patterns, like you said before...

Comment (pelikhan) [reply services/vibration.md:16]: yes!

Comment (pelikhan) [reply services/vibration.md:16]: and it should be finite. you don't want to have it vibrate forever becaues of a bug

Comment (pelikhan) [reply services/vibration.md:16]:
```
{
repeats:
    intensity: u8 /
    duration: u8 8ms
}
```

**Comment by pelikhan** [comment]
@mmoskal I tweaked the service. speed is a reading, enabled is on/off and you have to use commands to launch "vibration" nimations.

**Review by mmoskal — APPROVED** [review]

**Pull request merged**